tools/nl/embeddings/finetuning/run.sh

```diff
@@ -13,29 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ERR=true
-if [ "$1" == "-f" ] || [ "$1" == "-a" ]; then
-  if [ "$1" == "-f" ] && [ "$2" != "" ]; then
-    STAGE="final"
-    TUNED_ALT_MODEL="$2"
-    ERR=false
-  elif [ "$1" == "-a" ]; then
-    STAGE="alternatives"
-    TUNED_ALT_MODEL=""
-    ERR=false
-  fi
-fi
+function help {
+  echo "Usage: "
+  echo "$0 -f # finetunes the existing finetuned intermediate model on prod (final stage only)"
+  echo "$0 -i # the complete finetuning procedure (both stages: intermediate -> final)" 
+}
+
+STAGE=""
+while getopts fi OPTION; do
+  case $OPTION in
+    f)
+        echo -e "### Finetuning an existing tuned_alternatives model (final stage only)"
+        STAGE="final"
+        ;;
+    i)
+        echo -e "### Starting the complete finetuning procedure (both stages: alternatives -> final)"
+        STAGE="intermediate"
+        ;;
+    *)
+        help
+    esac
+done
 
-if [ "$ERR" == true ]; then
-    echo "Usage pattern # 1: $0 -f <tuned_alternatives_model_path_on_gcs>  # finetunes an existing tuned_alternatives model (final stage only)"
-    echo "Usage pattern # 1: $0 -a # the complete finetuning procedure (both stages: alternatives -> final)" 
+if [ "$STAGE" == "" ]; then
+  help
   exit 1
 fi
 
+# Get the model finetuned on sentence alternatives.
+PROD_FINAL_FINETUNED_MODEL=$(curl -s https://raw.githubusercontent.com/datacommonsorg/website/master/deploy/base/model.yaml | awk '$1=="tuned_model:"{ print $2; }')
```

Comment:
nit:  `| cut -f2- -d'.'` ?

Reply:
ah nice! i was doing downstream parsing but this makes more sense